tests/features/java/runtime.feature

```diff
@@ -35,3 +35,13 @@ Feature: Openshift OpenJDK Runtime tests
        | JAVA_DIAGNOSTICS | true                |
     Then container log should contain /deployments/undertow-servlet.jar Hello from CTF test
       And container log should contain -XX:NativeMemoryTracking=summary
+
+  @openjdk
+  @ubi8/openjdk-8
+  @ubi8/openjdk-11
+  @ubi8/openjdk-17
+  @redhat-openjdk-18
+  @openj9
+  Scenario: Ensure JAVA_ARGS is no longer present in the run script
+    Given s2i build https://github.com/jboss-openshift/openshift-quickstarts from undertow-servlet
+    Then file /opt/jboss/container/java/run/run-java.sh should not contain JAVA_ARGS
```

Comment:
This is a static check for the contents of a file; we need to run a container and check the container log to see what was printed.